bats/tests/containers/published-udp-ports.bats

```diff
@@ -0,0 +1,42 @@
+load '../helpers/load'
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start container engine' {
+    start_container_engine
+    wait_for_container_engine
+}
+
+run_container_with_published_udp_port_and_connect() {
+    local ip=$1
+    local port=$2
+    local netcat_connect_addr=$3
+    ctrctl run -d --name socat-udp-"$port" -p "$ip":"$port":"$port"/udp alpine sh -c \
```

Comment:
The image (`alpine`) should be mirrored to GHCR; see https://github.com/rancher-sandbox/rancher-desktop/blob/main/bats/tests/helpers/images.bash